app/src/main/java/de/langerhans/odintools/tools/SettingsRepo.kt

```diff
@@ -0,0 +1,110 @@
+package de.langerhans.odintools.tools
+
+import de.langerhans.odintools.BuildConfig
+import javax.inject.Inject
+
+class SettingsRepo @Inject constructor(
+    private val executor: ShellExecutor,
+) {
+
+    init {
+        println("Settings init")
+        enableA11yService()
+        grantAllAppsPermission()
+        // Don't add to whitelist on debug builds, otherwise even Android Studio can't kill the app
+        if (!BuildConfig.DEBUG) {
+            addOdinToolsToWhitelist()
+        }
+    }
+
+    private fun enableA11yService() {
+        val currentServices =
+            executor.executeAsRoot("settings get secure enabled_accessibility_services")
```

Comment:
Use a KEY_ constant for this key as well?

Reply:
Sounds good!